features/step_definitions/networking.rb

```diff
@@ -1705,3 +1705,15 @@
     })
   end
 end 
+
+Given /^the IPsec is disabled on the cluster$/ do
```

Comment:
this step is only valid for versions before 4.15, guess no other case for 4.15+ will use this step

Reply:
Looks good to me @weliang1 Thanks. i hope you have tested various wait timers for NO status change as they tend to break sometimes

Reply:
> Looks good to me @weliang1 Thanks. i hope you have tested various wait timers for NO status change as they tend to break sometimes

@anuragthehatter Correct, `Given I wait up to xxxxx seconds for the steps to pass` is used in many places to wait for operators back to function after enabling/disabling IPsec